pages/index.js

```diff
@@ -1,22 +1,65 @@
-import Head from "next/head";
 import styled from "styled-components";
+import DateDisplay from "../components/DateDisplay";
+import Task from "../components/Task";
+import Welcome from "../components/Welcome";
 
 export default function Home() {
+  function getCurrentDate() {
+    const date = new Date();
+    const year = date.getFullYear();
+    const month = date.getMonth() + 1;
+    const dates = date.getDate();
+    const arr = [
+      "Sunday",
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+    ];
+    const day = date.getDay();
+    return year + "." + month + "." + dates + " " + arr[day];
+  }
   return (
-    <div>
-      <Head>
-        <title>My App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <Main>
-        <h1>My App</h1>
-      </Main>
-    </div>
+    <Wrap>
+      <WrapMask>
+        <WelcomeStyle>
+          <Welcome />
+        </WelcomeStyle>
+        <DateDisplayStyle>
+          <DateDisplay date={getCurrentDate()} />
+        </DateDisplayStyle>
+        <Task />
+        <Task />
+      </WrapMask>
+    </Wrap>
   );
 }
 
-const Main = styled.main`
-  text-align: center;
+const Wrap = styled.div`
+  width: 100vw;
+  min-height: 100vh;
+  background: url(/images/bgImage.png);
+  backdrop-filter: blur(2px);
+`;
```

Comment:
This looks the same as before

Reply:
@Roland-Hufnagel before was `height: 100vh`, and now is `min-height: 100vh`

Reply:
Okay. I thought the problem was the aspect ratio. But now I saw your background-image an the aspect ratio may not be a problem. Fine.